src/Files.App/ResourceDictionaries/PathIcons.xaml

```diff
@@ -1,4 +1,439 @@
+
+			<Style x:Key="New" TargetType="ContentControl">
```

Comment:
I think this key and the keys below could be a bit more descriptive, "New" doesn't tell much about its purpose.

Reply:
Also, is the target type correct? I would expect this to be OpacityIcon.

Reply:
Either one works

Reply:
How about starting the key with `ColorIcon...`?

Reply:
It's going to be renamed once we finish adding all the icons and we remove the existing ColoredIcon control.

Reply:
Yeah I think starting with ColorIcon or whatever the final name of the control will be is a good approach.

Reply:
I renamed the icons.